ruby/command-t/scanner/file_scanner.rb

```diff
@@ -81,35 +102,11 @@ def ensure_cache_under_limit
         @paths_keys << @path
       end
 
-      def path_excluded?(path, prefix_len = @prefix_len)
-        if apply_wild_ignore?
```

Comment:
Yep, but it got rolled into `Array#any` on an empty array which is very cheep. If we need to stick with `expand` I'll put it back.